src/Common/filesystemHelpers.cpp

```diff
@@ -280,14 +280,27 @@ time_t getModificationTime(const std::string & path)
     struct stat st;
     if (stat(path.c_str(), &st) == 0)
         return st.st_mtime;
-    DB::throwFromErrnoWithPath("Cannot check modification time for file: " + path, path, DB::ErrorCodes::PATH_ACCESS_DENIED);
+    DB::throwFromErrnoWithPath("Cannot modification change time for file: " + path, path, DB::ErrorCodes::PATH_ACCESS_DENIED);
```

Comment:
Btw, I'm not sure, but probably `CANNOT_STAT` is more suitable than `PATH_ACCESS_DENIED`